modules/ref_arch_setup/tasks/pe_versions.rb

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env ruby
+require "net/http"
+require "json"
+require_relative "../../ruby_task_helper/lib/task_helper.rb"
+
+# PE Versions Task using Ruby Task Helper
+class VersionsTask < TaskHelper
+  PE_VERSIONS_URL = "https://puppet.com/misc/version-history".freeze
+
+  # the minimum prod version supported by RAS
+  MIN_PROD_VERSION = "2018.1.0".freeze
+
+  def task(**_kwargs)
```

Comment:
what is up with the **_kwargs

Reply:
This is directly from the example:
https://github.com/puppetlabs/puppetlabs-ruby_task_helper

I was wondering the same thing when I saw it and found this explanation:
https://flushentitypacket.github.io/ruby/2015/03/31/ruby-keyword-arguments-the-double-splat-and-starsnake.html

"Keyword arguments give you a convenient shorthand for using hashes as an argument to a method. ... There is a new operator, the double splat, that is sort of the splat for keyword arguments."